circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableMetadataMirrorReplication.java

```diff
@@ -87,12 +83,19 @@ public void replicate() throws CircusTrainException {
       ReplicaLocationManager replicaLocationManager = new MetadataMirrorReplicaLocationManager(sourceLocationManager,
           TableType.PARTITIONED);
       sourceLocationManager.cleanUpLocations();
-      replica.updateMetadata(eventId, sourceTableAndStatistics, sourcePartitionsAndStatistics, sourceLocationManager,
-          replicaDatabaseName, replicaTableName, replicaLocationManager);
-      int partitionsCopied = sourcePartitions.size();
-
-      LOG.info("Metadata mirrored for {} partitions of table {}.{} (no data copied).", partitionsCopied, database,
-          table);
+      if (sourcePartitions.isEmpty()) {
+        // Replicate table metadata only
```

Comment:
Comment is a bit misleading, the LOG message is better documentation imo

Reply:
I've changed the message and put it in a log call